Docs on fallback (#8326)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/cloud-release-tracks.md

Choose the "Compatible" and "Extended" release tracks if you need a less-frequent release cadence, the ability to test new dbt releases before they go live in production, and/or ongoing compatibility with the latest open source releases of <Constant name="core" />.

### Using the Fallback release track

The **Fallback** release track provides an emergency rollback option for account admins if you suspect a regression in the "Extended" track. 

> Enable it by going to **Account settings** and clicking the **Fallback** release track button in the <Constant name="dbt_platform" /> interface, rather than through environment settings.

Reviewer: ```suggestion
1. Enable it by going to **Account settings** 
2. Click the **Fallback** release track button in the <Constant name="dbt_platform" /> interface, rather than through environment settings. 
3. Fill in the details in the **Revert to Fallback** pop up to confirm and share any info with dbt Support.```

> 

Reviewer: ```suggestion
Switching to **Fallback** alerts the dbt Support team, who may reach out to help resolve the issue. This track is meant only as a temporary safety option to unblock you and not for ongoing use. You should return to "Extended" or "Compatible" once the issue is resolved.
```

<DocCarousel slidesPerView={1}>

<Lightbox src="/img/docs/dbt-versions/rollback.png" width="80%" title="Fallback release track button in dbt platform"/>

<Lightbox src="/img/docs/dbt-versions/rollback-popup.png" width="80%" title="Fallback release track popup in dbt platform"/>

</DocCarousel>

> Switching to **Fallback** alerts the dbt team, who may reach out to help resolve the issue. This track is meant only as a temporary safety option to unblock you and not for ongoing use. You should return to "Extended" or "Compatible" once the issue is resolved.

Reviewer: ```suggestion
```

Choose the "Compatible" and "Extended" release tracks if you need a less-frequent release cadence, the ability to test new dbt releases before they go live in production, and/or ongoing compatibility with the latest open source releases of <Constant name="core" />.

### Using the Fallback release track

The **Fallback** release track provides an emergency rollback option for account admins if you suspect a regression in the "Extended" track. 

1. Enable it by going to **Account settings** 
2. Click the **Fallback** release track button in the <Constant name="dbt_platform" /> interface, rather than through environment settings. 

> 3. Fill in the details in the **Revert to Fallback** pop up to confirm and share any info with dbt Support.```

Reviewer: ```suggestion
3. Fill in the details in the **Revert to Fallback** pop up to confirm and share any info with dbt Support.
```
